### ISSUE TITLE
Breaks when not using simple_form

### ISSUE THREAD

**Issue by kristianmandrup** [xdite/bootstrap-helper#18]
I'm using formtastic, not simple_form.

When I include this gem in my Gemfile, it breaks with the following:

```
uninitialized constant SimpleForm
/Users/kmandrup/.rvm/gems/ruby-1.9.3-p125/bundler/gems/bootstrap-helper-fc4818d80dc5/config/initializers/simple_form.rb:2:in `<top (required)>'
```

You should wrap this piece of the code with `if defined?(SimpleForm)`.
I will make a fork and a pull with this fix ASAP.

**Comment by xdite** [comment]
thanks

**Comment by xdite** [comment]
as you wish, i published 2.1.2.1
